### DISCUSSION TITLE
Drools excel execution in Grule

### DISCUSSION THREAD

**Discussion by stank3496** [hyperjumptech/grule-rule-engine#325] [General]
Hi Team,

I'm looking for a way to execute rule sheet of Drools in go language using Grule library. Is anyone used excel rule sheet from Drools in Grule?

Thanks,
Sneha

**Comment by newm4n** [comment]
Hi @stank3496 ,

Thanks for checking out Grule.
The idea on implementing decision table based rule engine for Grule has been around for quite a while now.
This ticket still open and we're still looking for help (Support of DMN Notation or/and Decision Tables #167 )  

Ferdinand

**Comment by newm4n** [comment]
Btw. this project, https://github.com/global-soft-ba/decisionTable might be the one you need.
It tries to implement Table Based rule DMN notation. It uses Grule inside.

cheers
